net/chrony/Makefile

```diff
@@ -8,7 +8,7 @@
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=chrony
-PKG_VERSION:=4.5
+PKG_VERSION:=4.6
```

Comment:
I think you've got the PKG_VERSION and PKG_RELEASE changes backwards, should be the same version, but the release gets bumped for non-functional changes.

Reply:
> I think you've got the PKG_VERSION and PKG_RELEASE changes backwards, should be the same version, but the release gets bumped for non-functional changes.
> 
> EDIT: oops, should also add that PKG_VERSION is dictated by upstream...

Thanks for the hint, fixed